tachyon/math/elliptic_curves/bls12/bls12_curve.h

```diff
@@ -74,63 +75,65 @@ class BLS12Curve : public PairingFriendlyCurve<Config> {
 
     // f1 = f.CyclotomicInverseInPlace() = f^(q⁶)
     Fp12 f1 = f;
-    f1.CyclotomicInverseInPlace();
+    CHECK(f1.CyclotomicInverseInPlace());
```

Comment:
I also think that implementing a separate function for similar instances in test code might be nice. The main difference is that instead of `CHECK` we use `ASSERT_TRUE`.
For example in `tachyon/cc/math/finite_fields/prime_field_unittest.cc` we have:
``` 
  const std::optional<bn254::Fr> cc_a_inv = cc_a_.Inverse();
  const std::optional<tachyon::math::bn254::Fr> a_inv = a_.Inverse();
  ASSERT_TRUE(cc_a_inv && a_inv);
  EXPECT_EQ(c::base::native_cast((*cc_a_inv).value()), *a_inv);
```
and if we made the function
```

Reply:
After internal discussion, I've realized I can simply also use `unwrap` for unittest cases!